packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -396,6 +396,12 @@ class ScheduleManagerCore {
         for (const pending of allPending) {
             this.trackPending(pending);
         }
+
+        // We are intentionally directly listening to the "op" to inspect system ops as well.
+        // If we do not observer system ops, we are likely to hit 0x296 assert when system ops
```

Comment:
nit: observer -> observe.
The sentence "incomplete batch" in the last line seems out of place.